rider/gradle.properties

```diff
@@ -8,7 +8,7 @@ pluginSinceBuild = 193
 pluginUntilBuild = 203.*
 # Plugin Verifier integration -> https://github.com/JetBrains/gradle-intellij-plugin#plugin-verifier-dsl
 # See https://jb.gg/intellij-platform-builds-list for available build versions
-pluginVerifierIdeVersions = 2019.3.5, 2020.1.4, 2020.2.3, 2020.3
+pluginVerifierIdeVersions = RD-2019.3.4, RD-2020.1.4, RD-2020.2.3, RD-2020.3.2
```

Comment:
Will we need to release a new version of the plugin when a new version of Rider ships?

Reply:
Yes. 
Line 8 (`pluginUntilBuild`) has to updated for every new major version. (Or else the plugin will not show up when searching in Rider)
Line 7 (`pluginSinceBuild`) needs to be updated when we drop support for a version.
Line 11 "should" be updated accordingly, though this is "only" the list of versions we're doing automated api-call test against.